src/Nethermind/Nethermind.Consensus/TargetAdjustedGasLimitCalculator.cs

```diff
@@ -39,7 +39,7 @@ public long GetGasLimit(BlockHeader parentHeader)
 
             long? targetGasLimit = _miningConfig.TargetBlockGasLimit;
             long newBlockNumber = parentHeader.Number + 1;
-            IReleaseSpec spec = _specProvider.GetSpec(newBlockNumber);
+            IReleaseSpec spec = _specProvider.GetSpec(newBlockNumber, parentHeader.Timestamp); // taking the parent timestamp is a temprory solution
```

Comment:
I think we should have a general solution where we use parentHeader.Number + 1, maybe use the current time stamp?